db/schema.rb

```diff
@@ -10,9 +10,9 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 0) do
+ActiveRecord::Schema.define(version: 2023_10_21_222726) do
 
-  create_table "archive_configs", id: :integer, options: "ENGINE=MyISAM DEFAULT CHARSET=utf8", force: :cascade do |t|
+  create_table "archive_configs", id: :integer, options: "ENGINE=MyISAM DEFAULT CHARSET=utf8mb3 COLLATE=utf8mb3_general_ci", force: :cascade do |t|
```

Comment:
I realise `utf8` is an alias for `ut8mb3` so this is correct, and all that gets overridden anyway when an ODAP-produced database is loaded into the db. But since the Rails app is configured for `utf8mb4` it might be less confusing if `schema.rb` was too.

Reply:
Just did a migration to update this accordingly, also manually updated structure.sql